db/migrate/20230316050815_create_saved_yardsales.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSavedYardsales < ActiveRecord::Migration[7.0]
+  def change
+    create_table :saved_yardsales do |t|
+      t.integer :user_id
+      t.integer :yardsale_id
```

Comment:
Similar to the other tables. We may want this to be foreign keys.

Reply:
Also, Rails has a handy method to create this kind of join tables:

https://api.rubyonrails.org/v7.0.4.2/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_and_belongs_to_many